content/400-pulse/200-getting-started.mdx

```diff
@@ -10,6 +10,12 @@ toc: true
 
 ## Prerequisites
 
+<Admonition type="info">
+
+💡 Pulse supports PostgreSQL. We'd love to hear [which databases](https://tally.so/r/wLbb8G) you would like to see supported next.
```

Comment:
```suggestion
💡 Prisma Pulse currently supports PostgreSQL. We'd love to hear [which databases](https://tally.so/r/wLbb8G) you would like to see supported next.
```

Reply:
Thank you for the suggestion! I omitted ~~Prisma~~ Pulse because the context of Pulse is clear and left currently to have the sentence match that on Platform Console and there we're strapped for space :D